docker.Makefile

```diff
@@ -114,5 +114,13 @@ shellcheck: build_shell_validate_image
 	docker run -ti --rm $(ENVVARS) $(MOUNTS) $(VALIDATE_IMAGE_NAME) make shellcheck
 
 .PHONY: test-e2e
-test-e2e: binary
+test-e2e: test-e2e-non-experimental test-e2e-experimental
```

Comment:
Would be nice to have a `make help` in this makefile as well (definitely not for this PR though :smile:)